Roadmap entry: ckETH and ckERC-20 tokens

dietersommer opened the pull request:
Thank you for your contribution to the IC Developer Portal.
Before submitting your Pull Request, please make sure that:

- [ ] You have updated the [`.github/CODEOWNERS`](https://github.com/dfinity/portal/blob/master/.github/CODEOWNERS) file according to your changes.
- [ ] You have registered new documents to [`/sidebars.js`](https://github.com/dfinity/portal/blob/master/sidebars.js) if any.

manudrijvers commented:
Our plan is that we have ckETH in 2023 but we will do ckERC20 after that, and there I think 2023 for ckERC20 is optimistic. Maybe we can split it into two entries, where we put 2023 on ckETH, but something later on ckERC20?

dietersommer commented:
> Our plan is that we have ckETH in 2023 but we will do ckERC20 after that, and there I think 2023 for ckERC20 is optimistic. Maybe we can split it into two entries, where we put 2023 on ckETH, but something later on ckERC20?

Just split it into the features on the roadmap and adjusted the timeline to 2024 for ckERC-20.
